docs/website/root/manual/getting-started/run-signer-node.md

```diff
@@ -396,6 +396,7 @@ RestartSec=60
 User=cardano
 EnvironmentFile=/opt/mithril/mithril-signer.env
 ExecStart=/opt/mithril/mithril-signer -vvv
+PIDFile=/opt/squid/var/run/squid/squid.pid
```

Comment:
Did you mean something like /opt/mithril/mithril-signer.pid ?

I then looked at my existing service file for mithril and read the systemd reference at https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#PIDFile= which suggests against setting the PIDFile:

> "Note that PID files should be avoided in modern projects. Use Type=notify, Type=notify-reload or Type=simple where possible, which does not require use of PID files to determine the main process of a service and avoids needless forking."

After reading this do you still recommend setting PIDFile in the mithril-signer service file for a particular reason?

Reply:
This line is just a test currently as the PR is still a draft, so there is no specific recommendation yet :slightly_smiling_face: 

However, I guess that you are right and the Mithril signer service should not be modified, this line was intended for the Mithril relay service configuration.